server/citation/api.ts

```diff
@@ -27,29 +29,40 @@ const zoteroStyleKindMap: Record<CitationStyleKind, ZoteroStyleKind> = {
 
 // the parameters which can be passed to the get() method
 // can be found at https://www.zotero.org/support/dev/web_api/v3/basics
-app.get('/api/citations/zotero', (req, res) => {
-	const userId = req.user?.id;
-	const { q, include, style = 'apa' } = req.query;
-	const zoteroStyle = zoteroStyleKindMap[style as CitationStyleKind];
-	if (!userId) return new Error('Log in to request citations');
```

Comment:
this was the problem, the error was not thrown.

but if the error was thrown, it wouldn't have been handled properly because it's not `wrap`ed and it's just a generic error, so you get a ton of scary 500s